### PR TITLE
Support more chains + other small things

### PR DESCRIPTION
**Pull request by BobTheBuidler**
This PR will add support for:

Moonriver chain
Aurora chain
multicall state override for periods before multicall2 was deployed
autodetection of active brownie web3 connection

I also replaced a whitelisted multicall2 with one with no whitelist for Heco chain

### CODE REVIEW

File: multicall/constants.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
 from enum import IntEnum
 
+try:
+    from brownie import web3, network
+    if network.is_connected(): w3 = web3
```
Comment (banteg) [inline multicall/constants.py:5]: this should fall through to web3py if brownie network is not connected (not run from a brownie script)

File: multicall/multicall.py [hunk 2]
```diff
@@ -19,7 +18,7 @@ def __init__(self, calls: List[Call], _w3=None, block_id=None, require_success:
 
     def __call__(self):
         if self.require_success is True:
-            multicall_map = MULTICALL_ADDRESSES
+            multicall_map = MULTICALL_ADDRESSES if self.w3.eth.chainId in MULTICALL_ADDRESSES else MULTICALL2_ADDRESSES
```
Comment (banteg) [inline multicall/multicall.py:21]: `chain_id` pls, the camelCase will be deprecated

**Review by banteg — APPROVED** [review]

File: multicall/constants.py (hunk 1 above)
Comment (BobTheBuidler) [reply multicall/constants.py:5]: Fix'd

File: multicall/multicall.py (hunk 2 above)
Comment (BobTheBuidler) [reply multicall/multicall.py]: Fix'd

**Pull request merged**